feather/server/src/systems/tablist.rs

```diff
@@ -11,11 +11,17 @@ use uuid::Uuid;
 
 use crate::{ClientId, Server};
 
-pub fn register(systems: &mut SystemExecutor<Game>) {
+pub fn register(game: &mut Game, systems: &mut SystemExecutor<Game>) {
+    game.insert_resource(TablistHeaderFooter {
+        header: "{\"text\":\"\"}".to_string(),
+        footer: "{\"text\":\"\"}".to_string(),
+    });
```

Comment:
I don't think it's that important to have default values, because this isn't a vanilla feature. If anyone needs this, it would be better to use a plugin.

Reply:
Feather is not even close to be fully survival-compatible, so the only usage for Feather in production will be lobbies and mini-games. And there's 99% chance that everyone needs this feature. If someone wants more advanced features (like placeholders, animations), it would still be possible to override the resource by a plugin. I don't know why vanilla server doesn't have this feature, but lack of feature is not a good thing to copy.

Reply:
Then in what format and where in the config would you like it?

Reply:
> In what format

Just like motd, using `Text::from(str)`. Or there is `libcraft_text::markdown` module that allows writing rich text components in format `@red some text @on_hover @show_text @green some hover text`, but I don't see any existing usage or documentation for this format.

> Where

`[server]` section